apm-agent-plugins/apm-quartz/apm-quartz-common/pom.xml

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>apm-quartz-job-plugin</artifactId>
+        <artifactId>apm-quartz</artifactId>
         <groupId>co.elastic.apm</groupId>
         <version>1.38.1-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>quartz-common</artifactId>
+    <artifactId>apm-quartz-common</artifactId>
     <name>${project.groupId}:${project.artifactId}</name>
 
     <properties>
-        <version.quartz>2.3.1</version.quartz>
```

Comment:
[for reviewer] the common part was compiled with quartz 2.x and contained some code that was only used with 2.x plugin, however making sure the common part is compiled with 1.x only is safer.